src/main/java/org/gridsuite/study/server/service/NetworkMapService.java

```diff
@@ -133,6 +133,22 @@ public String getEquipmentMapData(UUID networkUuid, String variantId, String equ
         return equipmentMapData;
     }
 
+    public String getHvdcLineWithShuntCompensators(UUID networkUuid, String variantId, String hvdcId) {
+        UriComponentsBuilder builder = UriComponentsBuilder.fromPath(
+                DELIMITER + NETWORK_MAP_API_VERSION + "/networks/{networkUuid}/hvdc-lines/{hvdcId}/shunt-compensators");
+        if (!StringUtils.isBlank(variantId)) {
+            builder = builder.queryParam(QUERY_PARAM_VARIANT_ID, variantId);
+        }
+        String path = builder.buildAndExpand(networkUuid, hvdcId).toUriString();
+        String equipmentMapData;
+        try {
+            equipmentMapData = restTemplate.getForObject(networkMapServerBaseUri + path, String.class);
+        } catch (HttpStatusCodeException e) {
```

Comment:
if (HttpStatus.NOT_FOUND.equals(e.getStatusCode())) {
                throw new StudyException(EQUIPMENT_NOT_FOUND);
            } else {
                throw handleHttpError(e, GET_NETWORK_ELEMENT_FAILED);
            }

Reply:
we may want to get infos on an unexisting Hvdc line (can happen in a modification). Then the endpoint cannot return NOT_FOUND.

Reply:
ok